cmd/cluster-image-registry-operator/main.go

```diff
@@ -53,7 +54,7 @@ func main() {
 	watch(corev1.SchemeGroupVersion.String(), "Secret", namespace, 0)
 	watch(kappsapi.SchemeGroupVersion.String(), "Deployment", namespace, 0)
 	watch(appsapi.SchemeGroupVersion.String(), "DeploymentConfig", namespace, 0)
-	watch(regopapi.SchemeGroupVersion.String(), "OpenShiftDockerRegistry", namespace, 5)
+	watch(regopapi.SchemeGroupVersion.String(), "OpenShiftDockerRegistry", namespace, 5*time.Second)
```

Comment:
these values should be higher, like 10 minutes.

Reply:
@bparees it will break operator logic.

Reply:
@bparees it's not about finalizer. it's about objects re-creation and upgrade.

Reply:
We get updates when objects change, we get delete events when they are deleted, we can look for objects at startup.  What is your scenario that requires you to poll every 5s?